client/src/explorer_common/explorer_components.js

```diff
@@ -181,15 +181,20 @@ export const ExplorerNode = ({
                 }
               >
                 {_.isFunction(val_display)
-                  ? val_display(get_val(node), node)
```

Comment:
I'm guessing this was a mistake haha

Reply:
Good catch haha. I have no idea why I even changed that line. It must have been by mistake.

That explanation makes more sense, thank you! I was just now playing with it trying to understand what it was doing 😅